packages/headless-plugin-ext/src/common/headless-plugin-ext-api-contribution.ts

```diff
@@ -0,0 +1,58 @@
+// *****************************************************************************
+// Copyright (C) 2023 EclipseSource and others.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License v. 2.0 which is available at
+// http://www.eclipse.org/legal/epl-2.0.
+//
+// This Source Code may also be made available under the following Secondary
+// Licenses when the conditions for such availability set forth in the Eclipse
+// Public License v. 2.0 are satisfied: GNU General Public License, version 2
+// with the GNU Classpath Exception which is available at
+// https://www.gnu.org/software/classpath/license.html.
+//
+// SPDX-License-Identifier: EPL-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0
+// *****************************************************************************
+
+import { PluginManager } from '@theia/plugin-ext';
+import { RPCProtocol } from '@theia/plugin-ext/lib/common/rpc-protocol';
+
+export * from '@theia/plugin-ext';
+
+declare module '@theia/plugin-ext' {
+/**
+ * Plugin API extension description.
+ * This interface describes scripts for all three plugin runtimes: frontend(WebWorker), backend(NodeJs), and headless(NodeJs)
```

Comment:
While this comment is true, the reader is likely going to scratch their head. It's not obvious that what's described here is the result of [interface merging](https://www.typescriptlang.org/docs/handbook/declaration-merging.html#merging-interfaces). The other properties of the interface come from a different module, specifically pulled in via line 20. I would add a comment here to help the reader make that connection.